docs/about-filecoin/primer.md

```diff
@@ -0,0 +1,147 @@
+---
+
+# Filcoin primer
+
+The following “syllabus” of instructional videos and materials covers all aspects of the Filecoin network – from mining to storage to application development. This resource is designed for anyone interested in learning about the history and development of the network, as well as how to leverage the network at all levels: mining, building, and storing.
+
+## Filecoin fundamentals
+
+### [History of the Filecoin project](https://www.zeroknowledge.fm/106) and [The Benefits of Decentralized Storage](https://www.youtube.com/watch?v=aW6CZh3EKSw)
+
+Protocol Labs Founder and CEO Juan Benet joined the teams at _Hackernoon_ and _ZeroKnowledge_ to talk about the history and future of Filecoin, IPFS, and Web3. These two interviews offer perspective on the Filecoin project, including how it has evolved since its inception and where it’s headed.
+
+Length: 2 hours
+
+Supplemental Materials:
+
+- [Filecoin: A Decentralized Storage Network](https://filecoin.io/filecoin.pdf) (2017 whitepaper)
+- [The Future of Decentralized Social Media with Balaji Srinivasan, Vitalik Buterin, Juan Benet](https://www.youtube.com/watch?v=DTxE9KV3YrE&list=PLXzKMXK2aHh43-aCLLuGnny7nhnsf49LO&index=7&t=9s)
+- [ProtoSchool: Decentralized Data Structures](https://proto.school/data-structures/)
+
+### [How Filecoin works](https://www.youtube.com/watch?v=P28aNAdZDi4)
+
+This workshop is a conceptual overview of how Filecoin works, including background on the project’s mission, a spec system overview, the key players in the network, and an overview of the Web3 ecosystem.
+
+Length: 2 hours
+
+Supplemental Materials:
+
+- Watch the [3D Filecoin Explainer](https://filecoin.io/) (click “Explore the filecoin vision”)
+- [Power Fault Tolerance - Technical Report](https://research.protocol.ai/publications/power-fault-tolerance/)
+- [Proof of Replication - Technical Report](https://filecoin.io/proof-of-replication.pdf)
+
+### [Intro to Filecoin mining](https://filecoin.io/blog/filecoin-guide-to-storage-mining/)
+
+There are two primary roles a node can play in the Filecoin network: storage and retrieval. In this session, Filecoin Software Engineer Łukasz Magiera explains how to mine on the Filecoin Network, walking through key concepts including storage sectors, storage proofs, and mining power. He offers a high-level overview of the network’s architecture and covers how to set up a mining node.
+
+Length: 1 hour
+
+Supplemental Materials:
+
+- [A Guide to Filecoin Storage Mining](https://filecoin.io/blog/filecoin-guide-to-storage-mining/)
+- [How to mine on the testnet](https://docs.filecoin.io/how-to/install-filecoin/)
+- [Troubleshooting mining issues](https://www.youtube.com/watch?v=D9qdv_3yNbM)
+- [Proof of Replication - Technical Report](https://filecoin.io/proof-of-replication.pdf)
```

Comment:
We have a how mining works version. I think supplemental materials should be linked from there. People will likely find it sooner than the "primer".